doc/src/Build_basics.rst

```diff
@@ -213,6 +213,13 @@ A few example command lines are:
 For compiling with the Clang/LLVM compilers a special CMake preset is
 included that can be loaded with `-C ../cmake/presets/clang.cmake`.
 
+In addition you can set ``CMAKE_TUNE_FLAGS`` to specifically add compiler
+flags to tune for optimal performance on given hosts. By default these are
+initialized to some compiler specific flags, where known, to optimize the
+LAMMPS executable with optimizations and instructions available on the host
+where LAMMPS is compiled. For example, for Intel compilers this would be
+``-xHost`` and for GNU compilers this would be ``-march=native``.
```

Comment:
Please mention, how would I turn this off? WIth `CMAKE_TUNE_FLAGS=OFF`?

Reply:
ok.

Reply:
done.